src/dtm/DTM.ts

```diff
@@ -20,6 +19,16 @@ import {
 } from './DTM_transport';
 import { DtmModulationMode, DtmPacketType, DtmPhysicalLayer } from './types';
 
+const validate0x09Command = (res: number[] | undefined) => {
+    if (!Array.isArray(res) || res.length !== 2 || (res[1] & 0x01) !== 0) {
+        throw new Error('Invalid result');
+    }
+    const rawDbmValue = (res[1] & 0xfe) >> 1;
+    const modifier = res[0] & 0x01 ? -128 : 0;
```

Comment:
maybe provide additional comment what modifier is? (if makes sense?)

Reply:
This is just two's complement. Maybe modifier is a bad name, but it's simply just checking whether we have the most significant bit set or not and therefore subtracting 128

Reply:
I also got confused with modifier ... put commit that we need 8 bits and then last bit is used in the validate hence we need the fist bit in res[0]